CODEOWNERS

```
@@ -0,0 +1,11 @@
+# Fallback on Flathub admins for unowned shared modules
+# TODO: What would be the right team?
+*                   @flathub/some-team
+
+dbus-glib/**        @TingPing
```

```suggestion
/dbus-glib/         @TingPing
```

Same for the others below. See https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax.

---

The `**` glob would have effectively been equivalent.

Good catch on the missing leading `/` though, that could have caused real issues for nested folders with the same names. :+1:

This is now fixed in the latest push to this branch.